### ISSUE TITLE
Questions Regarding the StockODE Model.

### ISSUE THREAD

**Issue by quangtn2606** [zxz4197/StockODE#1]
Dear Authors,

Thank you for your excellent paper and the accompanying source code. I appreciate the effort you’ve put into making your work accessible.

While reviewing the paper and code, I came across a few points that I would like to understand better:

Could you explain the process of generating hyper-edges based on sector-industry and wiki relations? Specifically, I noticed that you loaded the file [NYSE.txt](https://github.com/zxz4197/StockODE/blob/master/training/main.py#L58), which seems to merge both types of relations. Could you clarify how this merging is performed?

I observed that the "Movement Trend Correlation" block in the paper does not align with the implementation in the source code. In particular, the code uses a multiplication operation instead of employing a graph convolutional layer to aggregate explicit correlations among stocks [here](https://github.com/zxz4197/StockODE/blob/master/training/Trans_Graph_ODE.py#L264). Do you find this approach to be more effective than using GCN?

In the source code, different weights are used for loss calculation. For example, the loss function includes:
`
train_loss = cur_loss * self.parameters['b'] + self.parameters['a'] * kl_all + self.parameters['c'] * res_loss
`
Are these weights more effective than the method described in the paper?

I also noticed that the [kl_all](https://github.com/zxz4197/StockODE/blob/master/training/Trans_Graph_ODE.py#L291) loss tends to have a very high value. Could you explain your approach to minimizing this loss?

`kl_z = kl_divergence(z_distr, z_prior)`
`kl_z = torch.sum(kl_z)
`
`
kl_all = kl_z / float(1026)
`

Thank you for your time, and I look forward to your response!
